flink-connectors/flink-connector-hive/src/test/java/org/apache/flink/table/functions/hive/HiveSimpleUDFTest.java

```diff
@@ -122,29 +124,16 @@ public void testUDFJson() {
 	}
 
 	@Test
-	public void testUDFMinute() {
```

Comment:
why is this removed?

Reply:
UDFMinute isn't a class UDF any more, so the test is invalid in 3.1.x.